custom/tests.yaml

```
@@ -5188,6 +5188,22 @@ javascript:
             return {result: false, message: e.message};
           }
 
+svg:
+  elements:
+    a:
+      xlink_actuate: |-
+        var instance = document.createElementNS('http://www.w3.org/2000/svg', 'a');
+        instance.setAttributeNS('http://www.w3.org/1999/xlink', 'xlink:actuate', 'onRequest');
+        return !!instance && instance.getAttributeNS('http://www.w3.org/1999/xlink', 'actuate') === 'onRequest';
```

We should be able to just do this since the instance is already defined?  (Feel free to disregard this if I'm incorrect.)

```suggestion
      xlink_actuate: |-
        instance.setAttributeNS('http://www.w3.org/1999/xlink', 'xlink:actuate', 'onRequest');
        return !!instance && instance.getAttributeNS('http://www.w3.org/1999/xlink', 'actuate') === 'onRequest';
```

---

I removed these since they can now all be generated in the test builder thanks to the code you suggested above.